zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/adhocsubprocess/AdHocSubProcessActivityActivateProcessor.java

```diff
@@ -0,0 +1,243 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.engine.processing.adhocsubprocess;
```

Comment:
❓ @berkaycanbc, is this the correct package to place this processor in? It seems like the other packages within `io.camunda.zeebe.engine.processing` seem to represent the other `ValueType` variants, hence why we decided to place this processor in this package.